src/Cake.Core/Scripting/Processors/UriDirectiveProcessor.cs

```diff
@@ -6,12 +6,15 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
 using Cake.Core.Scripting.Analysis;
 
 namespace Cake.Core.Scripting.Processors
 {
     internal abstract class UriDirectiveProcessor : LineProcessor
     {
+        private Regex uriPrefixPattern = new Regex("(nuget|local|npm):");
```

Comment:
The `UriDirectiveProcessor` doesn't know anything about nuget, local or npm, so the regex should check for ANY scheme. Also, We generally makes members `readonly` and initialize them in the constructor.

Reply:
Maybe in the `FileLoadDirectiveProvider` the `file` should be handled?

Reply:
Something like `^([a-zA-Z]{2,}:)` would probably solve it. Windows drives can only be one character long and schemes that we accept has to be longer than one character.

Reply:
Thank you for the idea. I've just checked in the new pattern.